filters/include/pcl/filters/impl/frustum_culling.hpp

```diff
@@ -45,6 +45,11 @@
 template <typename PointT> void
 pcl::FrustumCulling<PointT>::applyFilter (Indices &indices)
 {
+    bool is_far_plane_infinite = (fp_dist_ == std::numeric_limits<float>::max());
+    if(is_far_plane_infinite) {
+        fp_dist_ = (np_dist_ != 0) ? np_dist_ : 1.0f;
```

Comment:
Wouldn't it be problematic to set near plane and far plane distance to the same value? Then the frustum would change to a 2D-rectangle. Wouldn't that cause problems later in the cross products? How about setting `fp_dist_ = np_dist_ + 1.0f`

Reply:
Do we need to consider the case when `np_dist_` is greater than std::numeric_limits<float>::max()-1?

Reply:
No, I would say that will never happen in practice